tractor/_streaming.py

```diff
@@ -609,7 +610,14 @@ async def open_stream(
                 # XXX: Make the stream "one-shot use".  On exit, signal
                 # ``trio.EndOfChannel``/``StopAsyncIteration`` to the
                 # far end.
-                await self.send_stop()
+                try:
+                    await self.send_stop()
```

Comment:
Also need a test for this:

- one end of a stream is closed 
- the other end trying to close before any other stream methods `.receive()`/`.send()` are called (thus detecting the closure before trying to send a stop msg)